store/storetest/store.go

```diff
@@ -6,12 +6,14 @@ package storetest
 import (
 	"github.com/stretchr/testify/mock"
 
+	"github.com/mattermost/mattermost-server/v5/services/cache"
 	"github.com/mattermost/mattermost-server/v5/store"
 	"github.com/mattermost/mattermost-server/v5/store/storetest/mocks"
 )
 
 // Store can be used to provide mock stores for testing.
 type Store struct {
+	Cache                     mocks.CacheProvider
```

Comment:
Not needed, the store itself doesn't need to know anything about the Cache

Reply:
Fixed